app/views/components/_image_meta.html.erb

```diff
@@ -1,12 +1,16 @@
 <%
   id ||= "image-meta-#{SecureRandom.hex(4)}"
   alt_text ||= nil
-  caption ||= nil
-  credit ||= nil
   actions ||= []
+  meta ||= []
   lead_image ||= false
+  no_border ||= false
+
+  if meta.any? && meta.key?(:alt_text)
```

Comment:
If you know better ways of checking for a hash key to exists please let me know and I'll update.

Reply:
you can use [`key?`](https://ruby-doc.org/core-2.5.1/Hash.html#method-i-key-3F)

note that https://github.com/alphagov/content-publisher/pull/280/files#diff-3ffed0748d3f0d0a0349498ec9bc47d7R5 sets this object to an array rather than a hash

Reply:
Hopefully you won't need `.any?` once this is initialised as a hash.